### PR TITLE
PHP 8.4 compatibility

### PR DESCRIPTION
**Pull request by aleswita**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea
```
Comment (xabbuh) [inline .gitignore]: this change should be reverted (you should add it to your local git ignore file instead)

File: phpunit.xml.dist [hunk 2]
```diff
@@ -8,6 +8,7 @@
         beStrictAboutTodoAnnotatedTests="true"
         failOnWarning="true"
         failOnRisky="true"
+        stopOnRisky="true"
```
Comment (xabbuh) [inline phpunit.xml.dist]: this should be reverted as well

Comment (aleswita) [reply phpunit.xml.dist]: OK, I added this line, because CI not showed deprecated warnings on PHP 8.4 CI run

**Review by xabbuh — APPROVED** [review]

**Review by jschaedl — APPROVED** [review]

**Comment by jschaedl** [comment]
Thank you @aleswita !

**Pull request merged**